docker/localnode/scripts/step1_configure_init.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env sh
+
+# Input parameters
+NODE_ID=${ID:-0}
+echo "Configure and initialize environment"
+# Set up GO PATH
+export GOPATH=$HOME/go
+export GOBIN=$GOPATH/bin
+export BUILD_PATH=/sei-protocol/sei-chain/build
+export PATH=$GOBIN:$PATH:/usr/local/go/bin:$BUILD_PATH
+
+# Prepare shared folders
+mkdir -p build/generated/gentx/
+mkdir -p build/generated/exported_keys/
+# Testing whether seid works or not
+./build/seid version # Uncomment the below line if there are any dependency issues
+# ldd build/seid
+
+# Initialize validator node
+MONIKER="sei-node-$NODE_ID"
+./build/seid init "$MONIKER" --chain-id sei >/dev/null 2>&1
+
+# Copy configs
+cp docker/localnode/config/app.toml ~/.sei/config/app.toml
+cp docker/localnode/config/config.toml ~/.sei/config/config.toml
+cp docker/localnode/config/price_feeder_config.toml ~/price_feeder_config.toml
+
+# Set up persistent peers
+SEI_NODE_ID=$(./build/seid tendermint show-node-id)
+NODE_IP=$(hostname -i | awk '{print $1}')
+echo "$SEI_NODE_ID@$NODE_IP:26656" >> build/generated/persistent_peers.txt
+
+# Create a new account
+ACCOUNT_NAME="node_admin"
+echo "Adding account $ACCOUNT_NAME"
+printf "12345678\n12345678\ny\n" | ./build/seid keys add "$ACCOUNT_NAME" >/dev/null 2>&1
+
+# Get genesis account info
+GENESIS_ACCOUNT_ADDRESS=$(printf "12345678\n" | ./build/seid keys show "$ACCOUNT_NAME" -a)
+echo "$GENESIS_ACCOUNT_ADDRESS" >> build/generated/genesis_accounts.txt
+
+# Add funds to genesis account
+./build/seid add-genesis-account "$GENESIS_ACCOUNT_ADDRESS" 10000000usei
+
+# Create gentx
+printf "12345678\n" | ./build/seid gentx "$ACCOUNT_NAME" 10000000usei --chain-id sei
+cp ~/.sei/config/gentx/* build/generated/gentx/
+
```

Comment:
not a blocker for this PR but could we also run the populate accounts script to create test accounts? They're useful for testing and it should be fairly quick 

https://github.com/sei-protocol/sei-chain/blob/master/scripts/old_initialize_local.sh#L39-L40

Reply:
Good point, added!